tracer/test/Datadog.Trace.OpenTracing.Tests/OpenTracingSpanTests.cs

```diff
@@ -90,8 +91,9 @@ public void Finish_StartTimeInThePastWithNoEndTime_DurationProperlyComputed()
 
             var otSpan = (OpenTracingSpan)span;
             var ddSpan = (Span)otSpan.Span;
-            double durationDifference = Math.Abs((ddSpan.Duration - expectedDuration).TotalMilliseconds);
-            Assert.True(durationDifference < 100);
+
+            var precision = TimeSpan.FromSeconds(1);
+            ddSpan.Duration.Should().BeCloseTo(expectedDuration, precision);
```

Comment:
Previously, I was getting a `durationDifference` of ~82ms and would occasionally fail being over 100ms, so a 1 second precision should be more than enough I'd imagine (maybe too much?)

Reply:
Sounds good to me, I think 1s is fine looking at the test👍